components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -1,10 +1,50 @@
 use tidb_query_codegen::rpn_fn;
 
 use crate::codec::data_type::*;
+use crate::codec::mysql::{RoundMode, DEFAULT_FSP};
 use crate::codec::{self, Error};
 use crate::expr::EvalContext;
 use crate::Result;
 
+#[rpn_fn]
+#[inline]
+fn abs_int(arg: &Option<Int>) -> Result<Option<Int>> {
```

Comment:
Would you mind extract `abs` as a trait `Abs` and implement for structure like `AbsInt` just like `Ceil` trait below.

Reply:
No problem, do I need to do the same thing with the `RoundXXX` and `RoundWithFracXXX` function?

Reply:
PTAL

Reply:
Great!